src/api/user/session.md

```diff
@@ -15,15 +15,15 @@ Get the current session used to authenticate.
 ```sh
```

Comment:
I wouldn't be surprised if I was running on at least 3 days of no sleep at this point when I wrote this specific file.